cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreMetadataTestRun.java

```diff
@@ -91,7 +112,28 @@ public void testGetTables() throws Exception {
                           ),
                           Lists.newArrayList(
                             new QueryResult(Lists.<Object>newArrayList(
-                              "", "default", "other_table", "TABLE", "CDAP Dataset")))
+                              "", NAMESPACE_DATABASE, otherTableName, "TABLE", "CDAP Dataset")),
+                            new QueryResult(Lists.<Object>newArrayList(
+                             "", OTHER_NAMESPACE_DATABASE, namespacedOtherTableName, "TABLE", "CDAP Dataset")))
+
+    );
+
+    // Pattern on database
+    future = getExploreClient().tables(null, OTHER_NAMESPACE_ID.getId(), "%", null);
+    assertStatementResult(future, true,
+                          Lists.newArrayList(
+                            new ColumnDesc("TABLE_CAT", "STRING", 1, "Catalog name. NULL if not applicable."),
+                            new ColumnDesc("TABLE_SCHEM", "STRING", 2, "Schema name."),
```

Comment:
Nope. Not a typo. I've not actually understood how Hive abbreviates properties though... It seems arbitrary.